firefox-webext/background.js

```diff
@@ -2,9 +2,98 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+var fetchURL = 'https://api.github.com/repos/webcompat/web-bugs/issues?state=open&page=1&per_page=100';
+var issuesList = [];
+var requestIssues = [];
+var ignoreList = ['about'];
+var ts = Math.round(new Date().getTime() / 1000);
+var tsRefresh = 0;
 var prefix = 'https://webcompat.com/?open=1&url=';
 var screenshotData = '';
 
+function addRefresh(now) {
```

Comment:
Can you comment on what the refresh is for? To avoid making GitHub API mad? To get fresh issues data?